pyproject.toml

```diff
@@ -0,0 +1,4 @@
+# File to configure the black formatter
+[tool.black]  
+line-length = 65 
```

Comment:
Where is the 65 coming from?

Reply:
i was also wondering about the line length, but from the alignment of the code snippet in pdf, this length compliant the current style, that can be the reason they  chose 65. pinging here @abhishekchitwar , as he implemented this.

Reply:
yeah, I have just use 65 as line length to accommodate the code snippet in PDF.

Reply:
Okay got it! Thanks for the explanation @abhishekchitwar